src/script/common/c_content.cpp

```diff
@@ -715,127 +715,176 @@ ContentFeatures read_content_features(lua_State *L, int index)
 	return f;
 }
 
-void push_content_features(lua_State *L, const ContentFeatures &c)
+void push_tiledef(lua_State *L, const TileDef &tiledef)
 {
-	std::string paramtype(ScriptApiNode::es_ContentParamType[(int)c.param_type].str);
-	std::string paramtype2(ScriptApiNode::es_ContentParamType2[(int)c.param_type_2].str);
-	std::string drawtype(ScriptApiNode::es_DrawType[(int)c.drawtype].str);
-	std::string liquid_type(ScriptApiNode::es_LiquidType[(int)c.liquid_type].str);
+	if (tiledef.backface_culling && tiledef.animation.type == TAT_NONE) {
+		lua_pushstring(L, tiledef.name.c_str());
+		return;
+	}
+
+	lua_createtable(L, 0, 3);
+
+	lua_pushstring(L, tiledef.name.c_str());
+	lua_setfield(L, -2, "name");
+	setboolfield(L, -1, "backface_culling", tiledef.backface_culling);
+
+	lua_createtable(L, 0, 3);
+
+	std::string type(es_TileAnimationType[(int)tiledef.animation.type].str);
+
+	lua_pushstring(L, type.c_str());
+    lua_setfield(L, -2, "paramtype2");
+
+	lua_createtable(L, 0, 3);
+	setintfield(L, -1, "aspect_w", tiledef.animation.vertical_frames.aspect_w);
+	setintfield(L, -1, "aspect_h", tiledef.animation.vertical_frames.aspect_h);
+	setfloatfield(L, -1, "length", tiledef.animation.vertical_frames.length);
+	lua_setfield(L, -2, "vertical_frames");
+
+	lua_createtable(L, 0, 3);
+	setintfield(L, -1, "frames_w", tiledef.animation.sheet_2d.frames_w);
+	setintfield(L, -1, "frames_h", tiledef.animation.sheet_2d.frames_h);
+	setfloatfield(L, -1, "frame_length", tiledef.animation.sheet_2d.frame_length);
+	lua_setfield(L, -2, "sheet_2d");
+
+	lua_setfield(L, -3, "animation");
+}
+
+void push_content_features(lua_State *L, const ContentFeatures &f)
+{
+	std::string paramtype(ScriptApiNode::es_ContentParamType[(int)f.param_type].str);
+	std::string paramtype2(ScriptApiNode::es_ContentParamType2[(int)f.param_type_2].str);
+	std::string drawtype(ScriptApiNode::es_DrawType[(int)f.drawtype].str);
+	std::string liquid_type(ScriptApiNode::es_LiquidType[(int)f.liquid_type].str);
 	
 	/* Missing "tiles" because I don't see a usecase (at least not yet). */
```

Comment:
It is already removed in my local version, but there are still some issues with tiles in this table. I will upload an updated version as soon as it works correctly. Thanks for your notice!